docs/user-guide/federation/federation-namespace.md

```diff
@@ -0,0 +1,90 @@
+---
+---
+
+This guide explains how to use events in Federation control plane to help in debugging
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general and [Namespaces](/docs/user-guide/namespaces/) in particular.
+
+## Overview
+
+Namespaces in federation control plane (refered to as "federation namespaces" in
+this guide) are very similar to the traditional [Kubernetes
+Namespaces](/docs/user-guide/namespaces/) providing the same functionality.
+Creating them in the federation control plane ensures that they are synced
+across all the registered clusters.
+
+
+## Creating a Federation Namespace
+
+The API for Federated Namespaces is 100% compatible with the
+API for traditional Kubernetes Namespaces. You can create a namespace by sending
+a request to the federation apiserver.
+
+You can do that using kubectl by running:
+
+``` shell
+kubectl --context=federation-cluster create -f myns.yaml
+```
+
+The '--context=federation-cluster' flag tells kubectl to submit the
+request to the Federation apiserver instead of sending it to a kubernetes
+cluster.
+
+Once a federation namespace is created, the federation control plane will create
+a matching namespace in all underlying kubernetes clusters.
+You can verify this by checking each of the underlying clusters, for example:
+
+``` shell
+kubectl --context=gce-asia-east1a get namespaces myns
+```
+
+The above assumes that you have a context named 'gce-asia-east1a'
+configured in your client for your cluster in that zone. The name and
+spec of the underlying namespace will match those of
+the Federation Namespace that you created above.
+
+
+## Updating a Federation Namespace
+
+You can update a federation namespace as you would update a Kubernetes
+namespace, just send the request to federation apiserver instead of sending it
+to a specific kubernetes cluster.
+Federation control plan will ensure that whenever the federation namespace is
+updated, it updates the corresponding namespaces in all underlying clusters to
+match it.
+
+## Deleting a Federation Namespace
+
+You can delete a federation namespace as you would delete a Kubernetes
+namespace, just send the request to federation apiserver instead of sending it
+to a specific kubernetes cluster.
+
+For ex, you can do that using kubectl by running:
+
+```shell
+kubectl --context=federation-cluster delete ns myns
+```
+
+As in Kubernetes, deleting a namespace will delete all resources in that
```

Comment:
Is this true?  If so, need to explain the corresponding resources in the underlying clusters are not deleted.